packages/host/app/components/matrix/room.gts

```diff
@@ -217,6 +227,13 @@ export default class Room extends Component<Signature> {
 
     return topMostCard;
   }
+
+  private get isSendButtonDisabled() {
+    return (
+      (!this.messageToSend && !this.cardsToAttach && !this.autoAttachedCard) ||
+      this.doSendMessage.isRunning
+    );
+  }
```

Comment:
based on my comment about not naming arguments to a component based on the interior state of elements in a component, perhaps a better name here is `canSend()`, since we are checking to see if we have the ability to send a message?

Reply:
yes, I was just looking at this. the need to know about presence of cards complicates it.